edbdeploy/terraform.py

```diff
@@ -141,21 +153,39 @@ def apply(self, vars_file):
 
     def destroy(self, vars_file):
         try:
-            rc = exec_shell_live(
+            # Added after Terraform 0.15.1
+            rc0 = exec_shell_live(
+                [self.bin("cd"), "%s" % self.dir],
+                environ=self.environ,
+                cwd=self.dir
+            )
+            if rc0 != 0:
+                raise Exception("Return code not 0")
+            # Added after Terraform 0.15.1                
+            rc1 = exec_shell_live(
+                [self.bin("terraform"), "init", "-no-color"],                
+                environ=self.environ,
+                cwd=self.dir
+            )
+            if rc1 != 0:
+                raise Exception("Return code not 0")
+            rc2 = exec_shell_live(
```

Comment:
1. No need to execute the `cd` command: the `cwd` argument (Change Working Directory) do that for you
2. I'm think we are not supposed to execute `terraform init` right before executing `terraform destroy`
3. Please pass the `TF_DATA_DIR` env. variable through the `environ` argument.